pkg/s3select/csv/args.go

```diff
@@ -30,7 +30,7 @@ const (
 	defaultRecordDelimiter      = "\n"
 	defaultFieldDelimiter       = ","
 	defaultQuoteCharacter       = `"`
-	defaultQuoteEscapeCharacter = `"`
+	defaultQuoteEscapeCharacter = `\\`
```

Comment:
As per csv, `"` is default quote escape character.  Have we decided to change it?

Reply:
Refer https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectSELECTContent.html

Reply:
We need to keep it like AWS S3 not like CSV

Reply:
The issue is perhaps we don't support anything other than `"` quote escape character